lib/firespring_dev_commands/templates/aws.rb

```diff
@@ -120,6 +120,57 @@ def create_eol_task!
           end
         end
       end
+
+      # rubocop:disable Metrics/MethodLength
+      # Create the rake task for the hosted zone method
+      def create_hosted_zone_task!
+        # Have to set a local variable to be accessible inside of the instance_eval block
+        exclude = @exclude
+
+        DEV_COMMANDS_TOP_LEVEL.instance_eval do
+          namespace :aws do
+            return if exclude.include?(:hosted_zone)
+
+            namespace :hosted_zone do
+              namespace :dns_logging do
+                task :init do
+                  @route53 = Dev::Aws::Route53.new
+
+                  domains = ENV['DOMAINS'].to_s.strip.split(',')
```

Comment:
I would opt for simplicity here - you should be able to only use the "domains" line. The split will just return an array with the single domain if there isn't a comma in the string.

I also wouldn't populate the one id array in the initializer. Once those pieces are removed I don't think the init is needed any more - I would just set things up in the tasks themselves.